discussion/evaluation-of-project.tex

```diff
@@ -1,6 +1,80 @@
-% !TEX root = ../main.tex
+q% !TEX root = ../main.tex
 
 \section{Project Evaluations}
 \label{sec:disc:project_evaluations}
 
-This section discusses the results that was gathered as part of the Project Tracker and the Circle Game projects, and presented in \autoref{chap:results}.
\ No newline at end of file
+This section discusses the results that was gathered as part of the Sensor Tracker the Circle Game projects, and presented in \autoref{chap:results}.
+First we will look at the choice of projects and method of evaluation.
+Then we will discuss each of the separated metrics, energy efficiency, performance and code size in isolation, before we wrap up this section by discussing the current state of Rust programming on an embedded system in light of our results.
+
+\subsection{Projects and methods}
+
+For the platform evaluation of this project we choose to develop two main example applications, one to evaluate performance and one for energy efficiency.
+These two \emph{qualitative} studies where chosen in order to focus on evolving the platform guided by these metrics.
+The reason for not choosing a more \emph{quantitative} study with a larger number of applications was due to the focus on developing the platform and that there does not exist a test suite written in Rust.
+The quantitative approach is a more common evaluation and would produce a larger set of results, providing a better foundation to evaluate the platform.
```

Comment:
"The qualitative approach is a more common evaluation.." 
Mer vanlig "evaluation" enn kvantitativ? Mer vanlig for hva?